SmartEVSE-3/src/evse.cpp

```diff
@@ -3121,45 +3113,6 @@ void StartwebServer(void) {
     },[](AsyncWebServerRequest *request, String filename, size_t index, uint8_t *data, size_t len, bool final){
     });
 
-    webServer.on("/currents", HTTP_POST, [](AsyncWebServerRequest *request) {
```

Comment:
This completely removes the API /currents endpoint and I can't see it being re-added anywhere?

Reply:
the endpoints were already duplicated in code... I also saw I missed another one (/reboot).
I think it is the result of a merge gone wrong....

Reply:
Verhip, you're right. That's weird.